client/js/components/support/DpSupport.vue

```diff
@@ -79,6 +82,9 @@ export default {
   },
 
   methods: {
+    hasPermission() {
+      return hasPermission
+    },
```

Comment:
Do you really need this one?

Reply:
tried - it was not necessary and got removed

Reply:
thx